ProGetAutomation/Functions/Remove-ProGetAsset.ps1

```diff
@@ -2,15 +2,15 @@ function Remove-ProGetAsset
 {
     <#
         .SYNOPSIS
-        Used to remove assets from the proget asset manager. 
+        Removes assets from ProGet. 
 
         .DESCRIPTION
-        This function removes assets from the proget asset manager. A session, assetName and assetDirectory is required. 
+        The `Remove-ProGetAsset` function removes assets from ProGet. A session, assetName and assetDirectory is required. 

```

Comment:
Remove the second sentence and add some documentation. Think of the documentation as instructions for how to use the function.

Reply:
Done